src/lib/requiredParams.ts

```diff
@@ -25,6 +25,7 @@ export const ADD_PROJECT = ["name", "customersId"] as const;
 export const ADD_SERVICE = ["name"] as const;
 export const ADD_TEAM = ["name"] as const;
 export const ADD_USER = ["name", "number", "email", "role"] as const;
+export const ADD_SURCHARGE = ["name", 'accumulation'] as const;
```

Comment:
```suggestion
export const ADD_SURCHARGE = ["name", "accumulation"] as const;
```

I'm surprised that you were able to commit an unformatted file 😁